### ISSUE TITLE
[object Object] appearing in Child link

### ISSUE THREAD

**Issue by brioapp** [perseids-tools/react-router-i18n#25]
Children route path:

```javascript
path: `${base}/posts/:id`
```

Single post link in posts component:

```javascript
<i18nLink
  to={{
    pathname: `/posts/${item.id}`,
    state: {
      title: item.title,
    },
  }}
>
```

Got link: 

```
http://localhost:3000/en/en/[object Object]
```

The correct link should be:

```
http://localhost:3000/en/posts/1
```

**Comment by brioapp** [comment]
Just tested again, when I removed properties from 'to', it works.

```
<Link to={`/posts/${item.id}`}>
```
it doesn't support 'to' as an object.

**Comment by zfletch** [comment]
Thanks for creating this issue, @brioapp! I've been able to replicate it and I created a [PR](https://github.com/perseids-tools/react-router-i18n/pull/26) that should fix it.

I've released the change in version `1.1.0-rc.1`. Could you confirm that, if you use that version, it fixes the issue?

**Comment by brioapp** [comment]
I've tested new version. the issue fixed, thanks for your work.

**Comment by brioapp** [comment]
I use `react-router-i18n` with `react-i18next` (handle dynamically fetched content translation). This package helps a lot in routing.

**Comment by zfletch** [comment]
Great, I've released version `1.1.0` with the fix. Thank you again for reporting the issue and for helping to test the solution!
